AsociacionDrupalES/DrupalCampSpain
# [#66] Set the new homepage and set block contents

We can't set block contents during site install because the
body field is not installed yet when dcamp_install() runs. Therefore,
I have updated README so it is now required to download the db
from Jenkins in order to install the site. This seems realistic to me
since now we have a production site so we need to work on database
updates from now on rather than installing a site.

## Testing steps

1. Download a copy of the production databse.
2. Install it.
3. Run database updates.
4. Open the homepage.

I see a couple bugs that we should work on follow ups though:

1. Block warnings during database updates:

```bash
juampy@Juampy/var/www/DCSpain/DrupalCampSpain/web(master)$ drush updb -y
The following module is missing from the file system: standard bootstrap.inc:236                                                           [warning]
The following updates are pending:

dcamp module : 
  8103 -   Add dummy sponsors for easy theming and default company logo. 
  8104 -   Add default blocks for frontpage. 
  8105 -   Sets the new theme and homepage. 

Do you wish to run all pending updates? (y/n): y
The following module is missing from the file system: standard bootstrap.inc:236                                                           [warning]
Performing dcamp_update_8103                                                                                                               [ok]
Performing dcamp_update_8104                                                                                                               [ok]
Performing dcamp_update_8105                                                                                                               [ok]
The block dcamp_2017_theme_breadcrumbs was assigned to the invalid region breadcrumb and has been disabled.                                [warning]
The block dcamp_2017_theme_help was assigned to the invalid region help and has been disabled.                                             [warning]
The block dcamp_2017_theme_messages was assigned to the invalid region messages and has been disabled.                                     [warning]
The block dcamp_base_theme_breadcrumbs was assigned to the invalid region breadcrumb and has been disabled.                                [warning]
The block dcamp_base_theme_help was assigned to the invalid region help and has been disabled.                                             [warning]
The block dcamp_base_theme_messages was assigned to the invalid region messages and has been disabled.                                     [warning]
Cache rebuild complete.                                                                                                                    [ok]
Finished performing updates.                                                      
```

2. Warning at Admin > Config:

![image](https://cloud.githubusercontent.com/assets/108130/23459082/35026976-fe7f-11e6-8dd5-2e8c9b4a7e2f.png)
